deploy/olm-catalog/portworx/23.5.0/portworx-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,540 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: portworx-operator.v23.5.0
+  namespace: placeholder
+  annotations:
+    capabilities: Auto Pilot
+    categories: "Storage"
+    description: Cloud native storage solution for production workloads
+    containerImage: portworx/px-operator:23.5.0-dev
+    repository: https://github.com/libopenstorage/operator
+    createdAt: 2023-03-21T21:29:41Z
+    support: Portworx, Inc
+    certified: "true"
+    operatorframework.io/initialization-resource: |-
+      {
+        "apiVersion": "core.libopenstorage.org/v1",
+        "kind": "StorageCluster",
+        "metadata": {
+          "name": "portworx",
+          "annotations": {
+            "portworx.io/is-openshift": "true"
+          }
+        }
+      }
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "core.libopenstorage.org/v1",
+          "kind": "StorageCluster",
+          "metadata": {
+            "name": "portworx",
+            "namespace": "test-operator",
+            "annotations": {
+              "portworx.io/is-openshift": "true"
+            }
+          },
+          "spec": {}
+        },
+        {
+          "apiVersion": "core.libopenstorage.org/v1",
+          "kind": "StorageNode",
+          "metadata": {
+            "name": "example",
+            "namespace": "test-operator"
+          },
+          "spec": {}
+        }
+      ]
+spec:
+  displayName: Portworx Enterprise
+  version: 23.5.0
+  minKubeVersion: "1.21.0"
+  maturity: stable
+  replaces: portworx-operator.v23.5.0
```

Comment:
Same as above. The value should be `23.3.1` for now until `23.4.0` is released.

Reply:
> Same as above. The value should be `23.3.1` for now until `23.4.0` is released.

I thought I did what you said -->

```
$ md5sum deploy/olm-catalog/portworx/23.4.0/core_v1_storagecluster_crd.yaml deploy/olm-catalog/portworx/23.5.0/core_v1_storagecluster_crd.yaml
743a94a183028d46fcd2a5c51d4a68d2  deploy/olm-catalog/portworx/23.4.0/core_v1_storagecluster_crd.yaml
743a94a183028d46fcd2a5c51d4a68d2  deploy/olm-catalog/portworx/23.5.0/core_v1_storagecluster_crd.yaml
```
* so, these 2 files are identical -- since I copied them